File: app/src/main/java/com/abona_erp/driver/app/data/model/DriverItem.java
package com.abona_erp.driver.app.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DriverItem {
  
  @SerializedName("LastnameFirstname")
  @Expose
  private String lastnameFirstname;
  
  @SerializedName("DriverNo")
  @Expose
  private Integer driverNo;
  
  @SerializedName("ImageUrl")
  @Expose
  private String imageUrl;
  
  public String getLastnameFirstname() {
    return lastnameFirstname;
  }
  
  public void setLastnameFirstname(String lastnameFirstname) {
    this.lastnameFirstname = lastnameFirstname;
  }
  
  public Integer getDriverNo() {
    return driverNo;
  }
  
  public void setDriverNo(int driverNo) {
    this.driverNo = driverNo;
  }
  
  public String getImageUrl() {
    return imageUrl;
  }
  
  public void setImageUrl(String imageUrl) {
    this.imageUrl = imageUrl;
  }
}

File: app/src/main/java/com/abona_erp/driver/app/data/model/DriverItem.kt
package com.abona_erp.driver.app.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class DriverItem (
    @SerializedName("LastnameFirstname")
    val lastnameFirstname: String?,

    @SerializedName("DriverNo")
    @Expose
    var driverNo: Int,

    @SerializedName("ImageUrl")
    @Expose
    var imageUrl: String?,

    @SerializedName("Sms")
    @Expose
    var sms: String?

)